src/js/app.js

```diff
@@ -14,6 +14,22 @@ export function registerListeners() {
       actions.prev();
     }
   });
+  const letterInputs = document.querySelectorAll('.word__char');
```

Comment:
We don't want the change from #75 in this PR. Could you revert this and I'll merge your added rebus 🙂